raiden/transfer/log.py

```diff
@@ -85,6 +85,30 @@ def __init__(self, database_path):
         )
         self.conn.commit()
 
+        self.sanity_check()
+
+    def sanity_check(self):
+        """ Ensures that NUL character can be safely inserted and recovered
```

Comment:
Looks good to me, just change add the missing 'L' in `NULL` here and in the `RuntimeError`